src/test/java/ro/msg/learning/shop/unit/strategy/AbstractStrategyTest.java

```diff
@@ -14,36 +13,54 @@ public abstract class AbstractStrategyTest {
 
     protected List<ProductQuantityDto> orderedProducts;
 
-    protected  Stock resultingStock1;
-    protected Stock resultingStock2;
-    protected List<Stock> resultingStockList1;
-    protected List<Stock> resultingStockList2;
-    protected List<StockDto> expectedResult;
-
+    protected Stock stock1;
```

Comment:
The naming of these objects is not intuitive, you should name the stocks in such a way that if I see the stock object in a test I should know what it is without coming here to see the value of it. Please adjust the names